README.md

```diff
@@ -165,6 +171,8 @@ Send POST request to localhost:<PORT>/serverControl
 -	schema - path to xsd schema file on mockserver classpath; default empty, so no vallidation of request is performed; if validation fails then response has got status 400 and response is raw message from validator
 -	imports - list of imports for closures (each import is separate tag); `alias` is the name of `fullClassName` available in closure; `fullClassName` must be available on classpath of mock server
 -   https - HTTPS configuration
+-   maxUses - limit uses of mock to the specific number, after that mock is removed (by default unlimited)
```

Comment:
add default as `-1` which means unlimited

Reply:
Nie pisałbym o tym, -1 to szczegół implementacji.

Reply:
So name it or set it in another way :) 
There always should be method to explicitly set value equal to default value in calls.

Reply:
Done.